app/views/contexts/summary/list_collector_block.py

```diff
@@ -71,9 +71,7 @@ def list_summary_element(self, summary: Mapping) -> dict:
         list_model = self._list_store[summary["for_list"]]
 
         list_collector_blocks = list(
-            self._schema.get_list_collectors_for_list(
-                self._section, for_list=summary["for_list"]
-            )
+            self._schema.get_list_collector_for_list(for_list=summary["for_list"])
         )
```

Comment:
This should fetch the list collector for the current section only?

Reply:
Updated 👍